lib/all-badge-examples.js

```diff
@@ -340,8 +347,9 @@ const allBadgeExamples = [
         title: 'Dockbit',
         previewUri:
           '/dockbit/DockbitStatus/health.svg?token=TvavttxFHJ4qhnKstDxrvBXM',
+        placeholderUri: '/dockbit/:organisation/:pipeline.svg?token=:token',
```

Comment:
Using `?token=:token` here renders this as `?token=%3Atoken` - any thoughts on a better style for this case?

Reply:
Heh, that's funny. My suggestion would be that we not URL-encode these. They aren't valid URLs, least I don't think they are; rather they are URL-like strings with placeholders. That justifies processing + rendering them in a different way.